### PR TITLE
update wfc3tools to 1.3.3

### PR DESCRIPTION
**Pull request by sosey**
Requesting to update the release package in astroconda to 1.3.3 for spacetelescope/wfc3tools

### CODE REVIEW

File: wfc3tools/meta.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ requirements:
     - scipy
     - python x.x
 source:
-    git_tag: 1.3.1
+    git_tag: 1.3.3
```
Comment (jhunkeler) [inline wfc3tools/meta.yaml]: Hey @sosey, I see you tagged `v1.3.3` against the wfc3tools repo. If you do that you need to prefix the `git_tag` version with `v` here in the recipe.

File: wfc3tools/meta.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ build:
     number: '0'
 package:
     name: wfc3tools
-    version: 1.3.1
+    version: 1.3.3
 requirements:
     build:
     - d2to1
```
Comment (jhunkeler) [inline wfc3tools/meta.yaml]: `d2to1` can be safely undefined now that you've moved to the astropy packaging framework ;)

File: wfc3tools/meta.yaml (hunk 1 above)
Comment (sosey) [inline wfc3tools/meta.yaml]: gah, the difference between relic and astropy still trips me, to be clean, would you rather me redo the git tag and notes or just update to the vtag

**Comment by sosey** [comment]
yah, was thinking that should be removed....will do

File: wfc3tools/meta.yaml (hunk 1 above)
Comment (jhunkeler) [inline wfc3tools/meta.yaml]: The "policy" I wrote in "Github and You" says you need to omit the `v` prefix while tagging, but it's more of uniformity/housekeeping issue than anything else. If we use `v` we then need to define the version twice in the recipe.

I need to update `wfc3tools` recipe to match the latest format. Take a look at https://github.com/astroconda/astroconda-contrib/blob/master/acstools/meta.yaml

^ that's what this should look like. I can take care of converting it tomorrow.

Comment (sosey) [inline wfc3tools/meta.yaml]: okay, I'll look tomorrow, and just fix it to be uniform. I remember the policy, but I forget when I'm working on astropy and stsci packages simulaneously, the v snuck in there, it was in the draft version notes and github tagged it when I published. I'll go check the draft hstcal notes to make sure it's off there too.

**Comment by sosey** [comment]
@jhunkeler Unsure of what exactly should be in the test commands section, is this directly related to the package tests or something else?

**Comment by jhunkeler** [comment]
@sosey `commands` are executed at the shell. Assuming `test_no_input` is a valid shell script or binary, it tests to ensure the command (a) exists in the environment's `PATH` post-install and (b) returns zero. Non-zero return values from *any* test will exit the build.

```yaml
commands:
    - my_utility --version       # no effect (if it works)
    - my_other_utility --version # no effect (if it works)
    - /bin/true                  # no effect
    - /bin/false                 # build will exit with failure
```

**Comment by pllim** [comment]
I view it as a place to test whether the executable scripts or entry points are properly installed. I don't see any executables installed by wfc3tools, so you can probably skip that part.

**Comment by sosey** [comment]
yes, I had already come to that conclusion, but the docs I read in conda weren't specific and I wasn't sure if joe was doing anything extra there.

**Comment by jhunkeler** [comment]
I only use them if I can. Some programs we produce don't exit cleanly so I don't include them in the `commands` section.

**Comment by sosey** [comment]
@jhunkeler yah, calwf3 (not built for this package, but called from this package), exits cleanly, but returns a syntax string when no arguments are presented.  Does my last commit fix the remaining issues?

**Comment by pllim** [comment]
IMHO calling `calwf3` here is out of scope. It should be done in HSTCAL meta.

**Comment by sosey** [comment]
That's why I'm NOT calling it here @pllim

**Comment by pllim** [comment]
Indeed. Just want to clarify it for everyone else who might look at this issue.

**Comment by jhunkeler** [comment]
I forgot to mention this last night. Sorry... bah!

Remove the `pip install d2to1 ...` line from `build.sh`

**Comment by sosey** [comment]
doh!  I removed it from both bld.bat and build.sh .... yes?

**Comment by jhunkeler** [comment]
aye ;)

**Comment by jhunkeler** [comment]
Builds fine on my end 👍 

🎅 merging

**Pull request merged**